Tests/LibJS/test-test262.cpp

```diff
@@ -172,7 +176,7 @@ class Test262RunnerHandler {
         }
 
         for (String const& line : lines) {
-            if (!m_output->write(String::formatted("{}\n", line)))
+            if (!m_output->write_or_error(String::formatted("{}\n", line).bytes()))
```

Comment:
Can we propagate errors here rather than using `write_or_error`?

Reply:
We can't here, failing to write might just be due to the process dying. 
Which is valid and we should just stop writing in that case.